packages/web/src/pages/track-page/components/desktop/RemixContestSection.tsx

```diff
@@ -0,0 +1,123 @@
+import { useRemixContest } from '@audius/common/api'
+import { useFeatureFlag } from '@audius/common/hooks'
+import { ID } from '@audius/common/models'
+import { FeatureFlags } from '@audius/common/services'
+import { UPLOAD_PAGE } from '@audius/common/src/utils/route'
+import {
+  Box,
+  Button,
+  Flex,
+  IconCloudUpload,
+  IconTrophy,
+  Text
+} from '@audius/harmony'
+
+import { useNavigateToPage } from 'hooks/useNavigateToPage'
+import { useRequiresAccountCallback } from 'hooks/useRequiresAccount'
+import useTabs from 'hooks/useTabs/useTabs'
+
+import { RemixContestDetailsTab } from './RemixContestDetailsTab'
+import { RemixContestPrizesTab } from './RemixContestPrizesTab'
+import { RemixContestSubmissionsTab } from './RemixContestSubmissionsTab'
+
+const messages = {
+  title: 'Remix Contest',
+  details: 'Details',
+  prizes: 'Prizes',
+  submissions: 'Submissions',
+  uploadRemixButtonText: 'Upload Your Remix'
+}
+
+type RemixContestSectionProps = {
+  trackId: ID
+  isOwner: boolean
+}
+
+/**
+ * Section component that displays remix contest information for a track
+ */
+export const RemixContestSection = ({
+  trackId,
+  isOwner
+}: RemixContestSectionProps) => {
+  const navigate = useNavigateToPage()
+  const { data: remixContest } = useRemixContest(trackId)
+  const { isEnabled: isRemixContestEnabled } = useFeatureFlag(
+    FeatureFlags.REMIX_CONTEST
+  )
+  const isRemixContest = isRemixContestEnabled && remixContest
+
+  const tabs = [
+    {
+      text: messages.details,
+      label: 'details'
+    },
+    {
+      text: messages.prizes,
+      label: 'prizes'
+    },
+    {
+      text: messages.submissions,
+      label: 'submissions'
+    }
+  ]
+
+  const elements = [
+    <RemixContestDetailsTab key='details' trackId={trackId} />,
+    <RemixContestPrizesTab key='prizes' trackId={trackId} />,
+    <RemixContestSubmissionsTab key='submissions' trackId={trackId} />
+  ]
+
+  const { tabs: TabBar, body: TabBody } = useTabs({
+    tabs,
+    elements,
+    isMobile: false
+  })
+
+  const goToUploadWithRemix = useRequiresAccountCallback(() => {
+    if (!trackId) return
+
+    const state = {
+      initialMetadata: {
+        is_remix: true,
+        remix_of: {
+          tracks: [{ parent_track_id: trackId }]
+        }
+      }
+    }
+    navigate(UPLOAD_PAGE, state)
+  }, [trackId, navigate])
+
+  if (!trackId || !isRemixContest) return null
```

Comment:
lets also return null if it is a remix contest without a description. Even though this will eventually be required, id like old remix contests to be backwards compatible, since users will put the info in the track description.

Reply:
don't think the model has been updated yet. i'll add a TODO for now